e2e/support/commands/downloads/downloadUtils-untyped.js

```diff
@@ -103,12 +103,14 @@ const addCustomCommands = () => {
   Cypress.Commands.add("verifyDownload", verifyDownload);
 };
 
-module.exports = {
+const verifyDownloadTasks = {
+  isFileExist,
+  findFiles,
+};
+
+export {
```

Comment:
mmm not sure, typescript was complaining if I didn't do this but it didn't complaint about other files

Reply:
It's just weird that you `require("module")` where `module` is actually using `import/export` syntax. I'm not sure if there's any complication since I'm not too well-versed with our build tool settings. But I think it's just a matter of style.